gensim/parsing/preprocessing.py

```diff
@@ -92,6 +224,27 @@ def strip_non_alphanum(s):
 
 
 def strip_multiple_whitespaces(s):
+    r"""Takes string, removes repeating in a row whitespace characters (spaces, tabs, line breaks) from it
```

Comment:
Why is this docstring `r"""`?

Reply:
This is special case because we used `\n`, `\r` in `Examples` section, from this Sphinx goes mad. Solution: use raw string for this case.